llama_stack/providers/remote/inference/passthrough/passthrough.py

```diff
@@ -137,8 +137,33 @@ async def chat_completion(
 
         params = {key: value for key, value in params.items() if value is not None}
 
+        json_params = {}
+        from llama_stack.distribution.library_client import (
```

Comment:
seems awkward to import something from library_client in a provider. can we move this method to some util?

Reply:
PR is for debugging, @SLR722 is working on a fix PR.